sources/core/objects/turfs/MetalWall.h

```diff
@@ -1,26 +1,72 @@
 #pragma once
 
 #include "Turf.h"
+#include <bitset>
+#include <array>
 
 namespace kv
 {
 
+const std::array<Dir, 4> WALL_PROCESSING_DIRS = {Dir::NORTH, Dir::SOUTH, Dir::WEST, Dir::EAST};
+
+class WallBitMask
```

Comment:
I don't think this helper class is needed. It is possible to just create serialization functions just for `std::biset<4>`, so this class doesn't add much value.